.github/workflows/publish.yml

```diff
@@ -21,7 +24,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: olafurpg/setup-scala@v13
+      - uses: actions/setup-java@v2
+        with:
+          distribution: temurin
```

Comment:
Maybe `java-version: 8` as well?

Reply:
I believe you need to add `java-version` here as well.

Reply:
Done